### ISSUE TITLE
Edible events issue

### ISSUE THREAD

**Issue by rachelslurs** [PicoNetworks/wordpress-plugin#146]
The edible local events pages are having their content completely removed by the plugin. We don't want to gate this content.

**Comment by rachelslurs** [comment]
### How I reproduced bug locally:

1. Did a backup using Updraft of their plugins from staging, restored them to my local and activated both `The Events Calendar` and `The Events Calendar PRO` plugins.
2. Used the wordpress export tool export just their events and imported them to my local
![screen shot 2018-08-15 at 6 06 22 pm](https://user-images.githubusercontent.com/1870406/44176056-3cd6c400-a0b6-11e8-808f-a58420c7efd8.png)
3. View events without Pico plugin connected
4. View events with Pico plugin connected, notice how the content is missing

It looks like their theme is trying to run `the_content()` twice. one within the generic `page.php` template and another template (`single-event.php`) included as part of `the_content()` of the `page.php`, but that `single-event.php` template calls `the_content()` as well. Need to see whether this is the way the plugin is intended to be used (calling `the_content()` twice for each event) or whether they modified the plugin to behave differently from what the plugin developer intended.

**Comment by rachelslurs** [comment]
Turns out there were no modifications made to the plugin. This is the plugin as it was intended 😱 .

I ended up adding a check to see if the current page is an event, and if so, the only modification I make to `the_content` is I wrap it with `<div id="pico">$content</div>`. I also set `Pico.show_read_more_button: false` on the events view.

I'm wondering if I need to pass something to the API to treat it as blacklisted content, so if a person visits 3 events, it doesn't say they've read 3 locked articles. I'm pretty sure the API will treat this as post_type CUSTOM, and thus lockable if the tags are listed in the rule/plan.

Should we just trust the publisher to not whitelist any tags that apply to events? Or maybe I shouldn't set the tags and categories for the events content type, that way it's treated like an index? I know I worked on this on the API side, but I might need to bounce the logic off of someone to make sure I don't mess up our metrics with whatever workaround we come up with.
